conf/base/bblayers.conf

```diff
@@ -18,7 +18,7 @@ BBLAYERS += "${BSPDIR}/sources/meta-uav"
 BBLAYERS += "${BSPDIR}/sources/meta-ros"
 BBLAYERS += "${BSPDIR}/sources/meta-intel-aero"
 BBLAYERS += "${BSPDIR}/sources/meta-intel-aero-connectivity"
-BBLAYERS += "${BSPDIR}/sources/meta-intel-realsense"
+BBLAYERS += "${BSPDIR}/sources/meta-intel-aero-librealsense"
```

Comment:
There were a few other things that weren't happy but right now what is described on #61 is the last issue I'm battling with.